docs/contributing/docs-templates.md

```diff
@@ -235,6 +236,10 @@ Here's a template for a new recipe category:
 
 ### Recipe parts
 
+#### Overview link
+
+To make sure your recipe is linked from the overview page, you must add it to the appropriate category in [`recipes.md`](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/recipes.md). Otherwise it will be difficult for Gatsby users to find it, which isn't good for anyone!
```

Comment:
```suggestion
To make sure your recipe is linked from the overview page, you must add it to the appropriate category in [`recipes.md`](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/recipes.md). Otherwise, it will be difficult for Gatsby users to find it, which isn't good for anyone!
```